reverse_engineering/helpers/clusterHelper.js

```diff
@@ -522,6 +532,38 @@ const getSelectedCollections = async ({ cluster, data, logger, app }) => {
 	}, {});
 };
 
+/**
+ * @param {{
```

Comment:
I would also add a comment that this function allows to get a document with the correct fields order.